data/Templates/Hl7v2/Resource/_MedicationRequest.liquid

```diff
@@ -589,27 +589,27 @@
         "requester": {
             {% if ORC.12.Repeats[0] == null or ORC.12.Repeats[0] == "" -%}
                 {% if RXO.14.Repeats[0] -%}
-                    {% evaluate practitionerRole_ID_RXO_14 using 'ID/practitionerRole' XCN: RXO.14.Repeats[0] -%}
+                    {% evaluate practitionerRole_ID_RXO_14 using 'ID/PractitionerRole' XCN: RXO.14.Repeats[0] -%}
                     "reference":"PractitionerRole/{{ practitionerRole_ID_RXO_14 }}",
                 {% elsif RXE.13.Repeats[0] %}
-                    {% evaluate practitionerRole_ID_RXE_13 using 'ID/practitionerRole' XCN: RXE.13.Repeats[0] -%}
+                    {% evaluate practitionerRole_ID_RXE_13 using 'ID/PractitionerRole' XCN: RXE.13.Repeats[0] -%}
                     "reference":"PractitionerRole/{{ practitionerRole_ID_RXE_13 }}"
                 {% elsif ORC.21.Repeats[0] %}
-                    {% evaluate practitionerRole_ID_ORC_21 using 'ID/practitionerRole' XCN: ORC.21.Repeats[0] -%}
+                    {% evaluate practitionerRole_ID_ORC_21 using 'ID/PractitionerRole' XCN: ORC.21.Repeats[0] -%}
```

Comment:
Is there a reason to not take this PR with these changes? https://github.com/microsoft/FHIR-Converter/pull/501/files

Reply:
No reason, I wasn't aware of this PR before I had already added this change, since it was required for my tests to pass.

Reply:
Okay, if possible it would be good to accept the original PRs so the original submitters get credit for their efforts.